pkg/auth/handler/oauth/challenge.go

```diff
@@ -37,6 +37,14 @@ func (p *ChallengeRequest) Validate() []validation.ErrorCause {
 	return nil
 }
 
+var challengeValidator = validation.NewValidator("http://v2.skgyear.io")
+
+func init() {
+	challengeValidator.AddSchemaFragments(
+		ChallengeRequestSchema,
```

Comment:
I think we need to remove `$id` from the schema fragment?

Reply:
Will do it as part of removing old JSON schema library.